### PR TITLE
add libtommath

### PR DESCRIPTION
**Pull request by wheybags**
Again, would like a repo under hunter-packages.
I added a cmake build system and have opened a pull request, but will still need a fork to provide a release of v1.0 that contains the cmake build system (also, they might reject my pr).

### CODE REVIEW

**Comment by ruslo** [comment]
> would like a repo under hunter-packages

https://github.com/hunter-packages/libtommath

**Comment by wheybags** [comment]
updated to the hunter-packages repo

File: cmake/projects/tommath/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cacheable)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    tommath
+    VERSION
+    "1.0-p1"
```
Comment (ruslo) [inline cmake/projects/tommath/hunter.cmake]: Same as [here](https://github.com/ruslo/hunter/pull/639/files#r103096124), can you please make it `v1.0-hunter`?

**Comment by wheybags** [comment]
renamed as well

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1376
* https://travis-ci.org/ingenue/hunter/builds/205494065

**Comment by ruslo** [comment]
Some tests failed, please follow https://github.com/ruslo/hunter/pull/639#issuecomment-280970213

**Comment by wheybags** [comment]
fixed in vs2008, also https://github.com/ingenue/hunter/pull/53

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1388
* https://travis-ci.org/ingenue/hunter/builds/205535905

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1389
* https://travis-ci.org/ingenue/hunter/builds/205645141

**Comment by ruslo** [comment]
Released:
* https://github.com/ruslo/hunter/releases/tag/v0.18.5

**Pull request merged**